deid/config/utils.py

```diff
@@ -284,9 +284,11 @@ def derive_ctp_coordinate(raw):
     if len(new_coordinate) != 4:
         bot.exit("Coordinates are expected to have length of 4, found %s" % raw)
     xmin, ymin, width, height = new_coordinate
+    new_coordinate[2] = xmin + width
+    new_coordinate[3] = ymin + height
 
     # Translate CTP coordinate to the convention we use
-    return [xmin, ymin, xmin + width, ymin + height]
+    return ",".join([str(i) for i in new_coordinate])
```

Comment:
Do we want to return a string here? Coordinates should technically be numbers. I think it would make sense to convert to string only when we need that, or perhaps have the default return as numbers, and have an argument that asks for a string format.

Reply:
It depends. :) 

Returned as a list of ints, this new function does not work with the existing deid code for coordinates.  When I started testing with this, specifying ctpcoordinates ended up with an exception ('int' object has no attribute 'lower') because the output of this function is eventually passed into [clean_pixel_data](https://github.com/pydicom/deid/blob/27d46bb8851952fc3c6639d4675e254907085af7/deid/dicom/pixels/clean.py#L329) which performs a .lower() on it - expecting that it's a string.

When I received this exception, I targeted the new code for a fix - rather than looking to refactor all of the other places where the code currently generates this string.

Reply:
Gotcha, and it looks like it's only used within that utils file:

```
$ grep -R derive_ctp_coordinate
Binary file deid/config/__pycache__/utils.cpython-39.pyc matches
deid/config/utils.py:def derive_ctp_coordinate(raw):
deid/config/utils.py:            coordinate = derive_ctp_coordinate(
deid/config/utils.py:            coordinate = derive_ctp_coordinate(
```
How about let's make it a private function (e.g., just rename to have a single underscore) and call it a day. We mostly don't want a user finding it and using it.

Reply:
Done!